src/USPSRateRequest.php

```diff
@@ -80,47 +112,54 @@ public function getRates() {
     return $rates;
   }
 
-  protected function initRequest() {
-    $this->usps_request = new Rate(
-      $this->configuration['api_information']['user_id']
-    );
-    $this->setMode();
-  }
+  /**
+   * Checks the delivery date of a USPS shipment.
+   *
+   * @return array
+   *   The delivery rate response.
+   */
+  public function checkDeliveryDate() {
+    $to_address = $this->commerce_shipment->getShippingProfile()
```

Comment:
**CHANGE REQUEST**

Have you actually tested after the latest code changes? The property was changed to `commerceShipment` both in its declaration and when it is set in `getRates`, but here it is `commerce_shipment`.